DOCS-1201: Update `ultrasonic` sensor docs and document `ultrasonic` camera (#1939)
viamrobotics/docs: docs/components/sensor/ultrasonic.md

weight: 60
type: "docs"
description: "Configure an ultrasonic model sensor."
tags: ["sensor", "components", "ultrasonic"]
icon: "/icons/components/sensor.svg"
images: ["/icons/components/sensor.svg"]
# SME: #team-bucket
---

{{< alert title="Tip" color="tip" >}}

> An ultrasonic distance sensor can also be configured as a [camera](/components/camera/) resource.
> [Configure it as such](/components/camera/ultrasonic/) to utilize the [camera API](/components/camera/#api) with your ultrasonic sensor hardware, enabling you to use it as an input to a [vision service](/services/vision/) model that returns obstacles.

Reviewer: I think this might be slightly better
```suggestion
An ultrasonic distance sensor can also be configured as a [camera](/components/camera/) resource.
When configured as a camera, you can use the [camera API](/components/camera/#api) and you can use the component as an input to a [vision service](/services/vision/) model that returns obstacles.
```

Author: Done!
